### DISCUSSION TITLE
Problem oc2 when loading

### DISCUSSION THREAD

**Discussion by BooleanFalse** [fnuecke/oc2#124] [General]
net.minecraftforge.fml.ModLoadingException: In OpenComputers II (oc2) an error was detected during the event phase done
§7net.minecraft.server.ChainedJsonException: Invalid shaders/core/projectors.json: Couldn't compile fragment program (Mod Resources, oc2:projectors)
: ERROR: 0:87: Invalid call of undeclared identifier 'texture2D'
ERROR: 0:88: Use of undeclared identifier 'projectorDepth'
ERROR: 0:90: Use of undeclared identifier 'projectorClipDepth'
ERROR: 0:91: Invalid call of undeclared identifier 'texture2D'
ERROR: 0:94: Use of undeclared identifier 'projectorColor'
ERROR: 0:101: Invalid call of undeclared identifier 'texture2D'
ERROR: 0:104: Use of undeclared identifier 'depth'
ERROR: 0:108: Use of undeclared identifier 'depth'
ERROR: 0:109: Use of undeclared identifier 'clipDepth'
ERROR: 0:110: Use of undeclared identifier 'worldPos'
ERROR: 0:115: Use of undeclared identifier 'worldPos'
ERROR: 0:115: Use of undeclared identifier 'worldNormal'
ERROR: 0:119: Use of undeclared identifier 'worldPos'
ERROR: 0:119: Use of undeclared identifier 'worldNormal'
ERROR: 0:123: Use of undeclared identifier 'worldPos'
ERROR: 0:123: Use of undeclared identifier 'worldNormal'

**Comment by fnuecke** [comment]
After a quick search, I'm guessing you're on MacOS?

**Reply by fnuecke** [reply]
Should be fixed in 0.1.6.

**Comment by BooleanFalse** [comment]
Yes, I use  MacOS. Understood. Thanks
